### DISCUSSION TITLE
Fbeta Score parameter setting

### DISCUSSION THREAD

**Discussion by siebe-hiel** [open-edge-platform/anomalib#1016] [Q&A] [answered]
Hi all,

I am trying to implement PaDiM and PatchCore using the FBetaScore in the torchmetrics collection. I can set FBetaScore in my config.yaml file, however I cannot seem to set the beta parameter there. Does anyone know how to fix this?

Thanks in advance!

**Comment by djdameln** [comment, marked as answer]
Hi, we currently don't support changing the default parameters of TorchMetrics classes from the config file. This is a limitation that we are planning to address in a future release. At this moment, your best option would be to implement a custom metric locally and add it to `anomalib.utils.metrics`. Your custom metric could inherit from TorchMetrics' FBetaScore and hard-code the value of the beta parameter. It's a bit inconvenient, but it would temporarily address your issue until we add a more advanced TorchMetrics support.
